Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_setup.rb

```diff
@@ -23,7 +22,8 @@
   "oi.dll", # Required for Import API
   "Polly.dll", # Required for Import API
   "Relativity.Logging.dll", # Required for Import API
-  "Relativity.Logging.Interfaces.dll" # Required for Import API
+  "Relativity.Logging.Interfaces.dll", # Required for Import API
+  "Relativity.Kepler.dll"

```

Comment:
I am not sure if this is the right solution. It seems like a workaround for today. This solution might not work in future where we would require a higher version of Relativity.Kepler as we are hardcoding the Kepler to version 2.7. 
@aarongilbert123 Can we discuss about this offline?

Reply:
discussed offline.